meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -66,11 +77,14 @@ export interface MongoPiece {
 	name: string
 	sourceLayerId: string
 	enable: {
-		start: number
+		start: number | string
```

Comment:
You propably have a good reason to make this type change, but i am curious as to why?